b-real-time/b-real-time-htb-system-tests.js

```diff
@@ -0,0 +1,191 @@
+'use strict';
+
+function getPartnerId() {
+    return 'BRealTimeHtb';
+}
+
+function getStatsId() {
+    return 'BRT';
+}
+
+function getCallbackType() {
+    return 'NONE';
+}
+
+function getArchitecture() {
+    return 'SRA';
+}
+
+function getConfig() {
+    return {
+        xSlots: {
+            1: {
+                tagid: 74072,
+                bidfloor: 0.01,
+                sizes: [[300, 250]]
+            },
+            2: {
+                tagid: 74072,
+                bidfloor: 0.01,
+                sizes: [[728, 90]]
+            }
+        }
+    };
+}
+
+function getBidRequestRegex() {
+    return {
+        method: 'POST',
+        urlRegex: /.*hb\.emxdgt\.com\/.?t=\d*&ts=\d*/
+    };
+}
+
+function validateBidRequest(request) {
+    var r = JSON.parse(request.body);
+    expect(request.body).toBeDefined();
+
+    expect(r.site.page).toBeDefined();
+    expect(r.site.domain).toBeDefined();
+
+    expect(r.imp.length).toBe(2);
+
+    expect(r.imp[0]).toEqual({
+        id: r.imp[0].id,
+        banner: {
+            format: [{
+                w: 300,
+                h: 250
+            }],
+            w: 300,
+            h: 250
+        },
+        tagid: 74072,
+        secure: r.imp[0].secure,
+        bidfloor: 0.01
+    });
+
+    expect(r.imp[1]).toEqual({
+        id: r.imp[1].id,
+        banner: {
+            format: [{
+                w: 728,
+                h: 90
+            }],
+            w: 728,
+            h: 90
+        },
+        tagid: 74072,
+        secure: r.imp[1].secure,
+        bidfloor: 0.01
+    });
+}
+
+function getValidResponse(request, creative) {
+    console.log(`NSC: getValidResponse -> request`, request.body);
```

Comment:
Please either remove this line or update it to use single-quote strings instead of template literals. The testing fails on IE otherwise.

Reply:
This has been removed, thanks for catching it.